translator/translate/logs/logs_collected/files/collect_list/collect_list_test.go

```diff
@@ -182,8 +236,8 @@ func applyRule1(t *testing.T, buf string) interface{} {
 	return val
 }
 
-// stdNumMonth     // "1"         //%-m
-// stdDay          // "2"         //%-d
+// stdNumMonth     // "_1"         //%-m
```

Comment:
This should be reverted.

Reply:
Correct must have forgot to remove it in the revision where I changed `timestamp_layout`